tests/unit_tests/test_text_splitter.py

```diff
@@ -26,3 +30,38 @@ def test_character_text_splitting_args() -> None:
     """Test invalid arguments."""
     with pytest.raises(ValueError):
         CharacterTextSplitter(chunk_size=2, chunk_overlap=4)
+
+
+def test_nltk_text_splitting_args() -> None:
```

Comment:
since these involve external dependencies (nltk/spacy) can move them to the integration tests folder?

Reply:
done